rpcd/playbooks/roles/rpc_maas/templates/rsyslogd_process_check.yaml.j2

```diff
@@ -0,0 +1,19 @@
+type: agent.plugin
+label: "rsyslogd_process_check--{{ ansible_hostname }}"
+disabled    : false
+period      : "{{ maas_check_period }}"
+timeout     : "{{ maas_check_timeout }}"
+details     :
+    file    : run_plugin_in_venv.sh
+    args    : ["{{ maas_plugin_dir }}process_check.py", "{{ rsyslogd_process_names|join(",") }}" {% if inventory_hostname in groups['all_containers'] %}, "-c", "{{ inventory_hostname }}" {% else %} {% endif %}]
```

Comment:
This should be:

```yaml
args    : ["{{ maas_plugin_dir }}process_check.py", "{{ rsyslogd_process_names|join(",") }}" {% if inventory_hostname in groups['all_containers'] %}, "-c", "{{ inventory_hostname }}" {% else %}{% endif %}]
```

Notice there's no space between `{% else %}` and `{% endif %}`.